### PR TITLE
added separated gulp tasks for performance scenarios (for dev speed);

### PR DESCRIPTION
**Pull request by VanDalkvist**

### CODE REVIEW

File: gulp/performance-tests.js [hunk 1]
```diff
@@ -58,3 +55,26 @@ gulp.task('performance-tests', ['all-tests-with-no-performance'], function () {
 		}, true)));
         */
 });
+
+var perfTestsPath = './performance-tests/tests';
+var files = fs.readdirSync(perfTestsPath);
+files.forEach(function (file) {
+	if (!fs.lstatSync(path.join(perfTestsPath, file)).isDirectory()) return;
+
+	gulp.task('performance-scenario-' + file, [], function () {
```
Comment (gerich-home) [inline gulp/performance-tests.js:64]: what will be the name of the task? smth like that: performance-scenario-computedDiamond1000UpdatesWith500Dependencies ?

Comment (VanDalkvist) [inline gulp/performance-tests.js:64]: no, `gulp performance-test-computedSubscription` or `gulp performance-test-computedDiamondUpdates`. By folder name

Comment (VanDalkvist) [inline gulp/performance-tests.js:64]: It is bulk of tests under concrete scenario

Comment (VanDalkvist) [inline gulp/performance-tests.js:64]: I didn't add all tests as the tasks

**Pull request merged**